tracer/src/Datadog.Trace/Generated/netstandard2.0/Datadog.Trace.SourceGenerators/TelemetryMetricGenerator/CiVisibilityMetricsTelemetryCollector_CountCIVisibility.g.cs

```diff
@@ -11,7 +11,7 @@
 namespace Datadog.Trace.Telemetry;
 internal partial class CiVisibilityMetricsTelemetryCollector
 {
-    private const int CountCIVisibilityLength = 610;
+    private const int CountCIVisibilityLength = 3055;
```

Comment:
wow, the combinatorial here has _exploded_ 😬 These are starting to be _very_ big arrays... I wonder when/if this will start to be a problem...

Reply:
Yeah I wonder if we can have an alternative simpler api even is slower, in CI Visibility I don't need blazing fast telemetry. I don't know if the amount of memory we are allocating here worth the improvements. For example the Go api has an array api for the tags. https://github.com/DataDog/dd-trace-go/blob/main/internal/civisibility/utils/telemetry/telemetry_count.go#L108-L113